src/components/modals/DarModal.js

```diff
@@ -43,71 +43,77 @@ const processResearchTypes = (researchTypes) => {
 };
 
 const requiresManualReview = (darDetails) => {
-  return darDetails.illegalBehavior ||
+  return (isNil(darDetails.requiresManualReview)) ?
+    (darDetails.illegalBehavior ||
     darDetails.poa ||
     darDetails.psychiatricTraits ||
     darDetails.sexualDiseases ||
     darDetails.stigmatizedDiseases ||
-    darDetails.vulnerablePopulation;
+    darDetails.vulnerablePopulation)
+    : darDetails.requiresManualReview;
 };
 
 const DarModal = (props) => {
   //NOTE: Modal should be simple (raw information should be passed in as props) in order to ensure plug and play use
   const {showModal, closeModal, darDetails, researcher} = props;
 
-  return h(Modal, {
-    isOpen: showModal,
-    onRequestClose: closeModal,
-    shouldCloseOnOverlayClick: true,
-    style: {
-      content: { ...Styles.MODAL.CONTENT },
-      overlay: {
-        backgroundColor: 'rgba(0, 0, 0, 0.5)'
+  if (!isNil(darDetails)) {
```

Comment:
Still going through the PR, here's a comment in the meantime...

Instead of using an if block here, use the ```isRendered``` conditional on ```DarModal```. This way the component won't render if the conditional evaluates to false.

Example: ```h(DarModal, {isRendered: !isNil(darDetails), ...```

Functional components in React are essentially render functions, so while you're not outputting anything, the component still initializes when it didn't have to.

That said, it could be confusing for a user to click the DAR name and not see a modal pop up when they were expecting it. Another possible approach is to output a simple modal that has a error statement like "DAR details were not provided" or something along those lines. Feel free to pick the option that best fits the situation.